Android/res/values-es/strings.xml

```diff
@@ -0,0 +1,454 @@
+<?xml version="1.0" encoding="utf-8"?>
+<resources>
+    <string name="pref_live_upload">Habilitar la subida en vivo (ALPHA TEST)</string>
+    <string name="Pause">Pausa</string>
+    <string name="about">Sobre</string>
+    <string name="account_label">Cuenta</string>
+    <string name="activity_title_map_provider_preferences">Preferencias del Proveedor de Mapas</string>
+    <string name="air_speed_label">Velocidad Aire</string>
+    <string name="altitude_label">Altitud</string>
+    <string name="app_settings">Opciones App</string>
+    <string name="area">Area</string>
+    <string name="button_scan">Buscar Dispositivos</string>
+    <string name="button_setup_calibrate">Calibrar</string>
+    <string name="button_setup_cancel">Cancelar</string>
+    <string name="button_setup_done">Hecho</string>
+    <string name="button_setup_next">Siguiente</string>
+    <string name="button_setup_reset">Reiniciar</string>
+    <string name="button_setup_send">Enviar</string>
+    <string name="camera_">Camara</string>
+    <string name="channel_label">Canal</string>
+    <string name="circle_radius">Radio Circular</string>
+    <string name="circle_turns">Numero de turnos</string>
+    <string name="climb_rate_label">Ratio de saltos</string>
```

Comment:
Indice de saltos? I would need a little context but the word Ratio should be changed.